cmd/init_project.go

```diff
@@ -186,16 +186,16 @@ func fetchAndCheckGoVersion() error {
 	cmd := exec.Command("go", "version")
 	out, err := cmd.Output()
 	if err != nil {
-		return fmt.Errorf("Failed to retrieve 'go version': %v", string(out))
+		return fmt.Errorf("failed to retrieve 'go version': %v", string(out))
```

Comment:
Note that all project use capital letters to do the outputs. So, why would be required do this change?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
So, it is a specific motivation and then IHMO should have a PR just for that where you would do this change across the repo. Note that in order to make clear what is done in the PR we are able to track the changes each PR should have a specific and clear motivation. So, wdyt about split it?